packages/itwin/map-layers/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "@itwin/map-layers",
+  "version": "4.0.0",
```

Comment:
make this a pre-release

Reply:
Sorry I dont understand what you mean, how can it be made pre-release?  Should I add a '-devXX' suffix?

Reply:
actually lets hold off on putting a version for this pkg, lets chat next week about this